integration-tests/environments/react-native/README.md

```diff
@@ -21,8 +21,9 @@ npx lerna bootstrap --scope @realm/react-native-tests --include-dependencies
 For iOS environments run 
 
 ```bash
+cd ios
```

Comment:
Are you use the first command should be run from "ios" too? It might traverse the filetree to look for the gemlock file 🤔

Reply:
`bundle install` was originally outside of the ios directory, but I asked if it should be inside instead due to React Native's [setup page](https://reactnative.dev/docs/environment-setup).

Reply:
Oh okay .. that works too then 😄